dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/jaxrs/model/AbstractResourceInfo.java

```diff
@@ -289,27 +383,37 @@ private void findContextFields(final Class<?> cls, Object provider) {
         return getProxyMap(SETTER_PROXY_MAP, create);
     }
 
-    private void findContextSetterMethods(Class<?> cls, Object provider) {
-
-        for (Method m : cls.getMethods()) {
-
-            if (!m.getName().startsWith("set") || m.getParameterTypes().length != 1) {
-                continue;
-            }
-            for (Annotation a : m.getAnnotations()) {
-                if (a.annotationType() == Context.class) {
-                    checkContextMethod(m, provider);
-                    break;
+    private void findContextSetterMethods(Class<?> cls, Object provider, ProviderContextInfo contextInfo) {
```

Comment:
same as above - can you surround these changes in `//Liberty change` comments?

Reply:
done